tcptracer-bpf.c

```diff
@@ -821,6 +841,48 @@ int kretprobe__inet_csk_accept(struct pt_regs *ctx)
 	return 0;
 }
 
+SEC("kprobe/fd_install")
+int kprobe__fd_install(struct pt_regs *ctx)
+{
+	u64 pid = bpf_get_current_pid_tgid();
+	u32 tgid = pid >> 32;
+	unsigned long fd = (unsigned long) PT_REGS_PARM1(ctx);
```

Comment:
Probably [here](https://github.com/iovisor/bcc/blob/381f03dbdb2d5723cb585c241bf05c6fcad8dd7a/src/cc/frontends/clang/loader.cc) and [here](https://github.com/iovisor/bcc/blob/381f03dbdb2d5723cb585c241bf05c6fcad8dd7a/src/cc/frontends/clang/b_frontend_action.cc#L281)